grml-hwinfo

```diff
@@ -496,10 +496,14 @@ cd "${OUTDIR}" || exit 1
           zpool list -vPL "$pool" > ./zfs/"$pool".list 2> ./zfs/"$pool".list.error &
           zpool status -vtiP "$pool" > ./zfs/"$pool".status 2> ./zfs/"$pool".status.error &
           zpool iostat -rp "$pool" > ./zfs/"$pool".histogram 2> ./zfs/"$pool".histogram.error &
-          zfs list -o space,referenced,logicalreferenced,quota,refquota -t filesystem,volume -r "$pool" > ./zfs/"$pool".zfslist 2> ./zfs/"$pool".zfslist.error &
+          zfs list -o space,referenced,logicalreferenced,quota,refquota \
+            -t filesystem,volume -r "$pool" > ./zfs/"$pool".zfslist 2> ./zfs/"$pool".zfslist.error &
           # machine parsable data
-          zpool get -Hp all "$pool" > ./zfs/"$pool".props 2> ./zfs/"$pool".props.error &
-          zfs get -Hp all "$pool" > ./zfs/"$pool".zfsprops 2> ./zfs/"$pool".zfsprops.error &
+          zpool get -p all "$pool" > ./zfs/"$pool".props 2> ./zfs/"$pool".props.error &
```

Comment:
Hm, the dropped "-H" isn't reflected in the commit message and this is a behavior change for users that might not be obvious?

I agree with @akorn's `My personal preference would be to keep the output human readable as opposed to machine readable by default` also. Having something machine parsable definitely makes sense, but that's what we get in the json output, nor? :thinking: